Modules/Product/Http/Services/ProductService.php

```diff
@@ -452,8 +561,10 @@ public function details($request): array
         $nextProductId = $nextProduct ? $nextProduct->id : 0;
         $brandDetails = Product::where('user_id', $products->user_id)->first();
         $bazaarDirectLink = $brandDetails->bazaar_direct_link;
-        $productImages = ProductImage::where('product_id', $request->id)->get();
-        $productVideos = Video::where('product_id', $request->id)->get()->toArray();
+
+        $productImages = Product::find($request->id)->productImages;
+
+        $productVideos = Product::find($request->id)->productVideos;
```

Comment:
Get all the relations using eager loading when you get the product like 
$product = Product::with('productImages'.'productVideos')where('user_id', $products->user_id)->first();
and the the relational data like 
$productImages = $product->productImages;
and same with the other relations.